### ISSUE TITLE
What to use while Window is being replaced

### ISSUE THREAD

**Issue by kalinkochnev** [imgui-rs/imgui-rs#657]
I cannot seem to create more than one window anymore. The `Ui` object seems to have disappeared the window() method.

I saw that the window API was being replaced, but I don't understand what we're supposed to use in the meantime. What is the last working version that supports windowing? And how different will the new API be? Will it be a massive pain to refactor if I switch to the new version once it's ready?

**Comment by kalinkochnev** [comment]
For anyone looking for a fix, I used `Window` and `ChildWindow` for all places where `ui.window()` was used before. That did the trick.
```rust
use imgui::{ChildWindow, Window};

Window::new("blah").build(ui, || {
    ChildWindow::new("blah").size([50f32, 0f32]).build(ui, ||{
        if ui.button(choices[value]) {
            value += 1;
            value %= 2;
        }

    });
    ui.show_demo_window(&mut true);
    ui.text_wrapped("Hello world!");
    // ui.text_wrapped("こんにちは世界！");
});
```

**Comment by dbr** [comment]
Your last comment is 100% correct for the currently published v0.8 crate.

Things are a little confusing currently because the last published version (0.8) doesn't match up with the current `master` branch (if you are looking for example code etc, you can find it by selecting the right tag in Github, [here](https://github.com/imgui-rs/imgui-rs/tree/v0.8.2))

To put differently, for the current published (v0.8) crate you would do:
```rust
imgui::Window::new("blah").build(ui, ||{
    ui.text("window");
});
```

The next version (v0.9) aims to make things consistent by creating windows etc from the `ui` object (same way the text is created), so you will do:

```rust
ui.window("blah").build(||{
    ui.text("contents");
});
```

> Will it be a massive pain to refactor if I switch to the new version once it's ready?

The changes between 0.8 and 0.9 should be a case of some "search and replace" fixes - so it should be pretty straight a straight forward, mechanical change (as in it shouldn't require any structural changes to your code, just slightly different syntax)

Basically replacing `Window::new("blah").build(ui, || {` with `ui.window("blah").build(|| {` (and similar for `ChildWindow` and others)

The `Window::new` method will have a deprecation warning pointing you to the new methods also
